packages/fireproof/src/database.ts

```diff
@@ -45,18 +45,18 @@ export class Database {
     })
   }
 
-  async get(id: string): Promise<Doc> {
+  async get<T extends Record<string, unknown> = {}>(id: string): Promise<Doc<T>> {
```

Comment:
this `T extends Record<string, unknown>` is repeating all over. Does it make sense to name it and reuse that. Maybe DocRecord?

Reply:
Yeah, absolutely makes sense.